### PR TITLE
complete changelog for v0.6.1

### PR DESCRIPTION
**Pull request by dunglas**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,8 +4,11 @@ CHANGELOG
 0.6.1
 -----
 
-* Add a new `passphrase` argument to `Symfony\Component\Mercure\Token\LcobucciFactory` allowing the use of encrypted keys.
-* Fixed a bug in `Symfony\Component\Mercure\Token\LcobucciFactory` that results in a runtime error when using "ecdsa" algorithms, alongside "lcobucci/jwt:^4.0".
+* Allow passing additional data to the `mercure` JWT claim when using `Symfony\Component\Mercure\Token\LcobucciFactory` 
+* Add a new `passphrase` argument to `Symfony\Component\Mercure\Token\LcobucciFactory` allowing the use of encrypted keys
+* Add a new `lastEventId` option to the `mercure()` Twig function to set the `Last-Event-ID` query paramet`r
```
Comment (azjezz) [inline CHANGELOG.md:9]:
```suggestion
* Add a new `lastEventId` option to the `mercure()` Twig function to set the `Last-Event-ID` query parametr
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,8 +4,11 @@ CHANGELOG
...
+* Fix a bug preventing setting cookies for legit subdomains when using ``Symfony\Component\Mercure\Authorization::createCookie()``
```
Comment (azjezz) [inline CHANGELOG.md:10]:
```suggestion
* Fix a bug preventing setting cookies for legit subdomains when using `Symfony\Component\Mercure\Authorization::createCookie()`
```

File: CHANGELOG.md (hunk 1 above)
Comment (chalasr) [reply CHANGELOG.md:9]:
```suggestion
* Add a new `lastEventId` option to the `mercure()` Twig function to set the `Last-Event-ID` query parameter
```
😅

**Review by azjezz — APPROVED** [review]
LGTM

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,8 +4,11 @@ CHANGELOG
 0.6.1
 -----
 
-* Add a new `passphrase` argument to `Symfony\Component\Mercure\Token\LcobucciFactory` allowing the use of encrypted keys.
-* Fixed a bug in `Symfony\Component\Mercure\Token\LcobucciFactory` that results in a runtime error when using "ecdsa" algorithms, alongside "lcobucci/jwt:^4.0".
+* Allow passing additional data to the `mercure` JWT claim when using `Symfony\Component\Mercure\Token\LcobucciFactory` 
+* Add a new `passphrase` argument to `Symfony\Component\Mercure\Token\LcobucciFactory` allowing the use of encrypted keys
+* Add a new `lastEventId` option to the `mercure()` Twig function to set the `Last-Event-ID` query parametr
```
Comment (chalasr) [inline CHANGELOG.md:9]: still not ^^
```suggestion
* Add a new `lastEventId` option to the `mercure()` Twig function to set the `Last-Event-ID` query parameter
```

**Pull request merged**